### PR TITLE
Create 2016-12-06.md

### PR DESCRIPTION
**Pull request by raynamharris**
@alixk and @gvwilson 
I thought it would be good to document some of our progress. Here is what I can recall from memory to serve as post hoc record of the task force's milestones. Did I forgot some milestones or links to relevant documents? Moving forward, we maybe can do a better job of keeping track of minutes from our meetings.

### CODE REVIEW

File: TaskForces/2017CarpentryCon/Minutes/2016-12-06.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# 2016-12-06 minutes (post hoc)
+Okay, tecnically these are not minutes as we did not have a meeting today. However, this is my attempt to reconcile the lack of minutes from previous meetings.
```
Comment (gvwilson) [inline TaskForces/2017CarpentryCon/Minutes/2016-12-06.md]: "technically" with an 'h'

**Comment by gvwilson** [comment]
One typo, the rest looks good - thanks!

File: TaskForces/2017CarpentryCon/Minutes/2016-12-06.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# 2016-12-06 minutes (post hoc)
+Okay, technically these are not minutes as we did not have a meeting today. However, this is my attempt to reconcile the lack of minutes from previous meetings.
+
+## 2016-12-05 progress update
+Working draft of Carpentry Con proposal sent to DC and SWC directors for comments.
+
+## 2016-2012-02 lack of progress update
```
Comment (wking) [inline TaskForces/2017CarpentryCon/Minutes/2016-12-06.md]: “2012” → “12”?

File: TaskForces/2017CarpentryCon/Minutes/2016-12-06.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+Today, Alix, Greg, and Rayna discussed weather or not we can announce a "save the date". We determined that we are NOT ready to share the date as we are far from confirming that the event has the interest/support/funding/ect. to move forward.
```
Comment (wking) [inline TaskForces/2017CarpentryCon/Minutes/2016-12-06.md]: “weather” → “whether”, although discussing the weather is fine too ;).

Comment (wking) [inline TaskForces/2017CarpentryCon/Minutes/2016-12-06.md]: Also  “ect” → “etc”.

**Pull request merged**